src/content/structured/components/badge/code.mdx

```diff
@@ -241,27 +241,27 @@ export const snippetsInformation = [
       />
     </svg>
   </IcBadge>
-  Success
+  Coffee Orders
 </IcButton>`,
   },
 ];
 
 <ComponentPreview snippets={snippetsInformation}>
   <div style={{ padding: "16px" }}>
     <IcButton variant="secondary" style={{ marginRight: "16px" }}>
-      <IcBadge text-label="1" slot="badge" variant="info" />
-      Number
+      <IcBadge text-label="8" slot="badge" type="text" variant="info" />
+      Coffee Orders
     </IcButton>
   </div>
   <div style={{ padding: "16px" }}>
     <IcButton variant="secondary" style={{ marginRight: "16px" }}>
       <IcBadge type="dot" slot="badge" variant="info" />
-      Dot
+      Coffee Orders
     </IcButton>
   </div>
   <div style={{ padding: "16px" }}>
     <IcButton variant="secondary" style={{ marginRight: "16px" }}>
-      <IcBadge type="icon" slot="badge" variant="info">
+      <IcBadge type="icon" slot="badge" text-label="new rosette" variant="info">
```

Comment:
I don't think this needs a text-label as it has `type="icon"` so the text won't show up anyway
```suggestion
      <IcBadge type="icon" slot="badge" variant="info">
```

Reply:
This will also apply to the code snippets too.